src/app/components/icons/wallet-type-ledger-icon.tsx

```diff
@@ -0,0 +1,13 @@
+import { BoxProps, Box } from '@stacks/ui';
+
+export function WalletTypeLedgerIcon(props: BoxProps) {
+  return (
+    <Box
+      as="img"
+      src="assets/images/wallet-type-ledger.svg"
```

Comment:
Fer recommended I export images from Figma as pngs at 2x. Not totally sure where this icon is showing up?

Reply:
It really depends on the image. Png is better for many of our illustrations, as there's lots of noise/detail that, when represented as a svg, would result in a large, inefficient vector.

<img width="205" alt="image" src="https://user-images.githubusercontent.com/1618764/173039227-50e1160f-cb51-4f32-b85f-ffc0beb4e085.png">

This image is a handful of rect/path svg elements, so there's no advantage to representing it as a png.